fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/api/event/client/HudRenderCallback.java

```diff
@@ -0,0 +1,16 @@
+package net.fabricmc.fabric.api.event.client;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface HudRenderCallback {
+	public static final Event<HudRenderCallback> EVENT = EventFactory.createArrayBacked(HudRenderCallback.class,
+		listeners -> (screenWidth, screenHeight, delta) -> {
+			for (HudRenderCallback event : listeners) {
+				event.render(screenWidth, screenHeight, delta);
+			}
+		}
+	);
+
+	void render(int screenWidth, int screenHeight, float delta);
```

Comment:
I know its basic but this needs docs. See something like the ToolTip event if you want inspriation.

Possibly explain what delta is or something?

Reply:
I am not very sure about the definition of delta, please check it out.